demo/gemini_conversation/app.py

```diff
@@ -190,13 +190,13 @@ def copy(self) -> "GeminiHandler2":
             gr.HTML(
                 """
                 <div style="display: flex; justify-content: center; align-items: center;">
-                <h1>Gemini Conversation</h1>                
+                <h1>Gemini Conversation</h1>
```

Comment:
Why did you delete this?

Reply:
If it is intentional I can add a noqa marker

Reply:
Ok it should be fine as is!